monai/handlers/parameter_scheduler.py

```diff
@@ -0,0 +1,163 @@
+import logging
+from bisect import bisect_right
+from typing import TYPE_CHECKING, Callable, Dict, List, Optional, Union
+
+from monai.utils import exact_version, optional_import
+
+if TYPE_CHECKING:
+    from ignite.engine import Engine, Events
+else:
+    Engine, _ = optional_import("ignite.engine", "0.4.4", exact_version, "Engine")
+    Events, _ = optional_import("ignite.engine", "0.4.4", exact_version, "Events")
+
+
+class ParamSchedulerHandler:
+    """
+    General purpose scheduler for parameters values. By default it can schedule in a linear, exponential, step or
```

Comment:
I think providing an example of usage would be of a great help for the users.

Reply:
I completely forgot about the use-case example. I will do another pull request to amend this one.